### PR TITLE
[CARBONDATA-753] Fix Date and Timestamp format issues

### PR DESCRIPTION
**Pull request by chenliang613**
Fix Date and Timestamp format issues:
1.Optimize the description of CARBON_TIMESTAMP_FORMAT&CARBON_DATE_FORMAT  in CarbonCommonConstants.java
2.Correct fields' definition of Date and Timestamp in examples.
3.Add example script how to show raw data's timestamp format. currently spark.sql.show() by default using "yyyy-mm-dd hh:mm:ss.fffffffff" as Timestamp.toString() format, users always wanting the show data be same as raw data format.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1040/

File: core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java [hunk 1]
```diff
@@ -383,7 +383,7 @@
    */
   public static final String CARBON_TIMESTAMP_DEFAULT_FORMAT = "yyyy-MM-dd HH:mm:ss";
   /**
-   * CARBON_TIMESTAMP_DEFAULT_FORMAT
+   * CARBON_TIMESTAMP_FORMAT,ONE PROPERTY FOR SPECIFYING DATATYPE IS TIMESTAMPE
```
Comment (jackylk) [inline core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java]: suggest to change to:
```
Property for specifying timestamp format for TIMESTAMP data type column. 
e.g. yyyy-MM-dd HH:mm:ss, or using CARBON_TIMESTAMP_DEFAULT_FORMAT
```

File: core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java [hunk 2]
```diff
@@ -392,7 +392,7 @@
    */
   public static final String CARBON_DATE_DEFAULT_FORMAT = "yyyy-MM-dd";
   /**
-   * CARBON_DATE_DEFAULT_FORMAT
+   * CARBON_DATE_FORMAT,ONE PROPERTY FOR SPECIFYING DATATYPE IS DATE
```
Comment (jackylk) [inline core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java]: suggest to change to:
```
Property for specifying date format for DATE data type column. 
e.g. yyyy-MM-dd, or using CARBON_DATE_DEFAULT_FORMAT
```

File: core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java (hunk 1 above)
Comment (chenliang613) [inline core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java]: How about "Property for specifying the format of TIMESTAMP data type column. 
e.g. yyyy/MM/dd HH:mm:ss, or using CARBON_TIMESTAMP_DEFAULT_FORMAT"

Comment (jackylk) [inline core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java]: ok

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1042/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1053/

**Comment by jackylk** [comment]
LGTM

**Pull request closed without merging**